adminapi/dataset/__init__.py

```diff
@@ -128,11 +135,12 @@ def commit(self, skip_validation=False, force_changes=False):
 
     def get_results(self):
         if self._results is None:
-            request_data = {
-                'filters': self._filters,
-                'restrict': self._restrict,
-                'order_by': self._order_by,
-            }
+            request_data = {'filters': self._filters}
+            if self._restrict is not None:
+                request_data['restrict'] = self._restrict
+            if self._order_by is not None:
+                request_data['order_by'] = self._order_by
```

Comment:
Is it ok to change the structure of the `request_data` variable?

Instead of
```
{
    'filters': self._filters,
    'restrict': None,
    'order_by': None
}
```
it will now just be:
```
{'filters': self._filters}
```